### DISCUSSION TITLE
Improve Support for BC

### DISCUSSION THREAD

**Discussion by LeCanardQuoi** [cds-snc/covid-alert-documentation#43] [Ideas]
# Summary | Résumé

As a user residing in British Columbia (and also a native French speaker), I wish to better understand 1) what is the app currently useful for,  2) what could be done in the short term to become more useful in the absence of support by the BC Health Ministry (ditto in Alberta) while adding additional pressure for the province to sign on, and 3) what actual requirements has the BC Health Ministry been seeking and never disclosed publicly (we have only been told that the app would not be “currently” useful back in October because the exposure notifications were too broad not sufficiently tied to specific gatherings, without any update whatsoever since).

# Design detail | Précisions au sujet de la conception

Revised app info should be incorporated within the app in lieu of the current “No reporting yet in your area” fatigued notice. The usefulness statement of currently turning on the app is misleading if trying to build trust toward a real BC support launch. Most users will immediately deactivate (using recently released feature) if there are no obvious benefits from the app, certainly when legitimately concerned with their mobile phone battery, other Bluetooth usage, apps clutter, marginal app adoption rate or perhaps even privacy concerns.  

In addition, the app should itself provide for more dynamic push notification when a new release is issued, rather than asking app users to configure for automatically updating the app or relying on app user to monitor the Playstore release info or social media notices.  As an example of current limited usefulness of the app , the number of diagnosed users currently reporting back (from other provinces) against which checks would be made is available under a Google setting log. The app user would be notified from these out of province visitors returned home (very marginal under current necessary travel only advisory).  Under cds-snc/covid-alert-app#1264 and cds-snc/covid-alert-app#1304, proximity contact info (not relying on the provincial health ministry support) would have preventative usefulness info for supporting mobile based social distancing. Under cds-snc/covid-alert-app#1305, reporting the localized app adoption rate would help increase confidence of the usefulness of the app, and better target deployment. Messaging script sent out by CDS support folks (in the ios and android app reviews) and various federal agencies&MPs is outdated and also needs revision.

This enhancement request thread could itself be used to record, track and schedule the specific requirements sought by the BC Health Ministry. Universal access via tokens (instead of mobile phones) cds-snc/covid-alert-app#1329 may be of interest for the objection once stated by the Health Minister. Support of QR codes for better handling diagnosed gatherings in public places may also be of interest cds-snc/covid-alert-app#1328. The active participation of the canadian advisory council for the app is sought, certainly some BC representation. 

For provinces that do not wish to sign on to supporting the app, over and above social distancing improvements proposed, an app diagnosed user should be able to notify a Canada wide server capable of triggering final notifications handling. Forcing the user to configure app for an other app supporting province seems an inappropriate "work-around". This would be best handled under a different enhancement request. 

Finally as a French native speaker, I would much prefer to be notified (hopefully faster) by the app configurable in French rather than by contract tracers struggling with my preference language on sensitive matters such as health. Other francophones in BC may also not be as fluent in English as I am. Other languages in BC (and elsewhere in Canada) with larger demographics could similarly benefit from improved multi-language support. This would also be best handled under a different enhancement request. 

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

Much of this enhancement request is also applicable to Alberta. A different enhancement request may be useful to track Alta specific requirements, such as interoperating with the ABTraceTogether app and “universal support” tokens cds-snc/covid-alert-app#1328 (available in the Singapore related implementation). All of the above should contribute to improving app adoption rates. 

The plan to increase the app adoption rates or relaunch in BC (in light of recent support of older ios versions and several proposed enhancements somewhat repurposing the app) has yet to be documented. It preferably should be accessible to the public via the github server. Pre-authorized notifications cds-snc/covid-alert-app#1347 could also have important impact when relaunching for BC.

Further thread discussions are clearly required,  preferably with the active participation of the canadian advisory council for the app.

**Comment by LeCanardQuoi** [comment]
Further thread discussions have not timely occured. Improved support for BC will most likely also require improvement in the process of managing new requirements such as better tracking/sign off of requirements of both BC Heath and Canada Health authorities discussed strictly privately so far (no public records readily available). https://github.com/cds-snc/covid-alert-server/issues/417 has been entered to track progress on managing new requirements and getting them allocated for development, testing and release -vs- other resources currently available or already allocated. More resources and budget may be necessary in order to timely deliver during the 2nd wave.

**Comment by kbroughton** [comment]
This is a public repo. I feel that if the BC gov were more public with their requirements, BC tech firms would be happy to contribute coding resources to solve these problems. It's the lack of transparency when journalists asks questions about their progress that is most upsetting. I would invite Dr Bonnie Hendry and Health Minister Adrian Dix to consider posting the requirements to this issue to improve the transparency of the process.

**Comment by LeCanardQuoi** [comment]
Mandatory BC business  COVID-19 safety plans promoted via QR code signage could be leveraged by linking to upcoming  QR code scan support cds-snc/covid-alert-app#1328 Thanks to Jeff D. for suggesting.
https://t.co/9UH9fqHgJD

**Comment by LeCanardQuoi** [comment]
Canadians living in BC, Alta, Yukon and the Nunavut would benefit from the app if it supported notifications bypassing the health authority and allowed notification of potential exposure (with proper disclaimer)  on the basis of a self-test kit.  cds-snc/covid-alert-documentation#38

**Reply by LeCanardQuoi** [reply]
Long overdue update - As BC is now experiencing its most serious pandemic hardship due to VOCs harmful spread (particularly P.1), extensive lockdowns are now looming in schools and workplaces due to airborne community transmission finally being reckoned with. Reopening would therefore have to eventually occur carefully. Improved tracing of new cases (that could not be done without adequate app based tooling) will be sought. Community herd will not be reached through vaccination for the next 6 months and preparing for a likely 4th wave in the fall may soon become required.  There are therefore great opportunities to start deploying QR code scanning capabilities, provide for self-test exposure notification (lessening current OTK stumbling block hindering exposure notification), aim toward a 75% app adoption rate, add trustworthy application info (such as covid test results, quarantine info, vaccine info) and enable contact tracing people *before* reopening. Provincial and regional health authorities may soon become more receptive to improved app based tooling to help us out of the currently deteriorating situation, as media coverage of our contact tracing tooling improves and more BCians start to actively seek meaningful changes in the management of the pandemic.

**Comment by obrien-j** [comment]
In keeping with our newly really release #31  practices, I've moved this up into the covid-alert-documentation repo as our central point for taking input from folks outside the team, and am converting the issue into a Discussion.
